save-cloud-charts/save-cloud/.gitignore

```diff
@@ -0,0 +1 @@
+/save-cloud-*.tgz
```

Comment:
I am making a weekly report now. Is this all the work you have done in this week? :D

Reply:
Why are you reviewing draft PRs?